### ISSUE TITLE
Narrator Does not Read Node.js Project Properties Select File Button Properly

### ISSUE THREAD

**Issue by mjbvz** [Microsoft/nodejstools#1361]
##### Steps to Reproduce
1. Turn on windows narrator 
2. Create a new node console app
3. Right click on the project in VS and go to properties.
4. Tab to one of the select file buttons, such as `Working Directory` or `Node.exe Path`

![image](https://cloud.githubusercontent.com/assets/12821956/19533189/9110e778-95f3-11e6-954e-0e1be59b6908.png)
##### Expected

Narrator makes it clear this button is used to select a file for the given setting.
##### Actual Behavior

Narrator reads:

> Button

Not so helpful.

**Comment by paulvanbrenk** [comment]
Fixed in Update 3
